<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Bwidget" name="BPlayopenGLWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>780</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Bopenfile_btn">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>556</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>打开文件</string>
    </property>
   </widget>
   <widget class="QSlider" name="BPlaySlider">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>527</y>
      <width>600</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="MediaPlayTime">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>530</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="MediaAllTime">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>530</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="BPlayLable">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>240</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Image.qrc">:/BPlay/Image/BPlay.png</pixmap>
    </property>
   </widget>
   <widget class="QSlider" name="BplayVolume">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>560</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="BplaySpeed">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>556</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>倍速</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>73</x>
      <y>560</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Image.qrc">:/BPlay/Image/Volume.png</pixmap>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Bwidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">bwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Image.qrc"/>
 </resources>
 <connections/>
</ui>
